broker-cli/proto/broker.proto

```diff
@@ -753,6 +753,8 @@ message MarketEvent {
 message GetOrderbookRequest {
   // Market to retrieve updates for, e.g. `'BTC/LTC'`.
   string market = 1;
+  // Number to limit results of either side of orderbook
+  int64 limit = 2;
```

Comment:
consider naming this something like `limit_per_side` or similar. It may be a bit strange to have a request that you pass a limit to that returns 2x the number you specify.